### PR TITLE
Implemented app final logic

### PR DESCRIPTION
**Pull request by Osong-Michael**
Finished implementing all functionality for the app to work

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @OA7 👋

**Good work so far on this.**

**You've done well so far. Just a few changes and we are good to go.** 

**STATUS: CHANGES REQUESTED.**

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/components/ButtonPanel.js [hunk 1]
```diff
@@ -1,37 +1,40 @@
...
+  ];
+
+  const orangeBtns = ['÷', 'x', '-', '+', '='];
+
+  const buttonList = buttonGroups.map(btnGrp => (
+    <div className="btn-grp" key={Math.random()}>
+      {
+        btnGrp.map(btn => (orangeBtns.includes(btn) ? <Button key={Math.random()} name={btn} handleClick={handleClick} /> 
+          : btn === '0' ? <Button key={Math.random()} name={btn} color="#E0E0E0" wide handleClick={handleClick} />
+            : <Button key={Math.random()} name={btn} color="#E0E0E0" handleClick={handleClick} />))
```
Comment (mikenath223) [inline src/components/ButtonPanel.js]: - [ ] Please avoid nesting ternary operators, it makes your code hard to read and understand.

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
-    <div className="btn-grp last-grp">
-      <Button name="0" color="#E0E0E0" wide />
-      <Button name="." color="#E0E0E0" />
-      <Button name="=" />
+  ));
+
+  return (
+    <div>
+      { buttonList}
     </div>
```
Comment (mikenath223) [inline src/components/ButtonPanel.js:32]: - Your code is still in review. Kindly exercise patience.

**Comment by mikenath223** [comment]
- [ ] This behavior could be confusing to a user consider removing the operator after the second digit is entered.
chrome-capture

**Comment by mikenath223** [comment]
- [ ] Rather than making a single commit you should have multiple commits helping to track changes.

**Comment by mikenath223** [comment]
- [ ] Please handle dividing by zero in a better way. Divisions by zero shouldn't return zero but rather should return an error.

File: src/components/ButtonPanel.js (hunk 1 above)
Comment (shloch) [reply src/components/ButtonPanel.js]: yeah, thia is not so cool

Comment (Osong-Michael) [reply src/components/ButtonPanel.js]: I know, I remember trying to use normal if conditions and it just gave me more headache

**Comment by shloch** [comment]
The operator issue above not taken care of...would be cool to do that

**Review by shloch — APPROVED** [review]
## Hello @OA7 

Great job with the improvements
You get a thumbs up for the efforts. 
calc2

Cool thing handling this division error

You can merge this

**Pull request merged**